### DISCUSSION TITLE
15 giugno 2022 - esercizio 3

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#13] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/7feb76bf-db46-47ff-8bd3-cb233bd16922)

**Comment by Alessandro700** [comment]
<img width="1578" height="995" alt="image" src="https://github.com/user-attachments/assets/830fce03-94e5-409c-94cb-a548c08f62e9" />
<img width="1122" height="274" alt="image" src="https://github.com/user-attachments/assets/e8799db8-ddeb-45e3-bc0a-f0076fe78d18" />

**Comment by M1SC0** [comment]
<img width="1296" height="579" alt="image" src="https://github.com/user-attachments/assets/550c8b41-8555-4727-a09e-437fa721478f" />
